wordle-web/components/SigninDialog.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <v-dialog v-model="modelValue" width="400">
+    <v-card>
+      <v-alert v-if="errorMessage" type="error" title-date-format="Function">
+        {{ errorMessage }}
+      </v-alert>
+      <v-card-text>
```

Comment:
You could have a v-sheet here since your Name dialog and help dialog use sheets to display titles. This goes the same for your add word dialog

Reply:
Thank you!